modules/openapi-generator/src/main/resources/C-libcurl/cJSON.c.mustache

```diff
@@ -841,12 +841,12 @@ static cJSON_bool print_string_ptr(const unsigned char * const input, printbuffe
     /* empty string */
     if (input == NULL)
     {
-        output = ensure(output_buffer, sizeof("\"\""));
+        output = ensure(output_buffer, sizeof("\"\"")+1);
```

Comment:
The souce code comes from cJSON https://github.com/DaveGamble/cJSON. we shouldn't hack to fix it here.
Suggest fixing it in cJSON project or leave this code as is.

Reply:
I'll see if they'll accept the changes.